homework-submission/my-project/src/AddForm.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import DatePicker from 'react-datepicker';
```

Comment:
You may have another look to the documentation of `react-datepicker` [Here exactly...](https://www.npmjs.com/package/react-datepicker#installation)
That it says you have to include `momentjs` in your application, And maybe another look to the [configuration section](https://www.npmjs.com/package/react-datepicker#configuration)